pkg/kev/config/svcextensions_test.go

```diff
@@ -127,7 +143,7 @@ var _ = Describe("Extentions", func() {
 				})
 
 				It("returns in defaults", func() {
-					k8sconf := config.DefaultK8SConfig()
+					k8sconf := config.DefaultSvcK8sConfig()
 					k8sconf.Workload.LivenessProbe.Type = config.ProbeTypeNone.String()
```

Comment:
Not strictly part of this PR but I think this case could be reworked slightly. Context specifies that we are dealing with missing replicas - but it's hard to see that context being prepared in BeforeEach. Also liveness probe is redundant information here?

Reply:
Will have a look.

Reply:
Fixed. IMO there are many BeforeEach / JustBeforeEach combinations making things a bit confusing.

Will aim to refactor those as I go along on new PRs.